lib/Rides.dart

```diff
@@ -313,67 +310,92 @@ class RideInProgressCard extends StatelessWidget {
       },
       child: DecoratedBox(
           decoration: BoxDecoration(
-              color: selected ? Color(0xFFBDBDBD) : Colors.white,
+              color: selected ? Color.fromRGBO(167, 167, 167, 0.4) : Colors.white,
```

Comment:
Like before, there seems to be a lot of switching between hex and fromRGBO. Is there any reason for that?

Reply:
Update: fixed in code cleanup PR